feature/security/gnsi/authz/tests/authz/authz1_4_test.go

```diff
@@ -285,9 +290,14 @@ func TestAuthz1(t *testing.T) {
 
 	t.Run("Authz-1.4, Test Normal Policy", func(t *testing.T) {
 		// Pre-Test Section
-		_, policyBefore := authz.Get(t, dut)
-		t.Logf("Authz Policy of the Device %s before the Rotate Trigger is %s", dut.Name(), policyBefore.PrettyPrint(t))
-		defer policyBefore.Rotate(t, dut, uint64(time.Now().Unix()), fmt.Sprintf("v0.%v", (time.Now().UnixNano())), false)
+		statusmsg, policyBefore := authz.Get(t, dut)
+		//t.Logf("Message for first authz get %s", msg)
+		if statusmsg == nil {
+			t.Logf("Expected error FAILED_PRECONDITION seen for authz Get Request.")
+		} else {
+			t.Logf("Authz Policy of the Device %s before the Rotate Trigger is %s", dut.Name(), policyBefore.PrettyPrint(t))
+			defer policyBefore.Rotate(t, dut, uint64(time.Now().Unix()), fmt.Sprintf("v0.%v", (time.Now().UnixNano())), false)
+		}
 
 		// Fetch the Desired Authorization Policy and Attach base Admin Policy Before Rotate
 		newpolicy, ok := policyMap["policy-normal-1"]
```

Comment:
it seems like this is part of a table-driven test setup, but the table was never setup?
Random magic strings like 'policy-normal-1' look like they belong int he test table data:
```golang 
  tests := []struct{
    descr string
    policy string
    want <whatever is wanted>
   wantErr bool
  }{{}}

  for _, test := range tests {
    // do the things
  }
```

or similar...